src/twisted/spread/test/test_pb.py

```diff
@@ -170,6 +169,52 @@ def disconnectClientFactory():
 
 
 
+@implementer(interfaces.IConnector)
+class ReconnectingConnector(object):
```

Comment:
Actually, there's already https://github.com/twisted/twisted/blob/c02c19441f6bd9d2431ac019ba36ac6abe33cfea/src/twisted/test/proto_helpers.py#L343.  Would it be too much trouble to refactor this to use that existing fake?